microservices/services/query-metric/api/src/main/resources/templates/querymetric.html

```diff
@@ -19,6 +19,10 @@
     <div th:utext="${header}"></div>
     <h1>Query Metrics</h1>
 
+    <div id ="system-name">
+        <p>Cluster: <span th:text ="${@environment.getProperty('datawave.query.metric.system-name')}"></span></p>
+    </div>
+
```

Comment:
Should we also include the system name in `/templates/querymetric-horizontal.html`?